tests/security/lynis/lynis_setup.pm

```diff
@@ -34,22 +34,24 @@ sub run {
 
     if (is_sle) {
         add_suseconnect_product("PackageHub", undef, undef, undef, 300, 1);
-        zypper_call("in lynis", timeout => 300);
     }
+    zypper_call("in lynis", timeout => 300);
 
     # Record the pkgs' version for reference
     my $results = script_output("rpm -qi lynis");
     record_info("Pkg_ver", "Lynix packages' version is: $results");
 
     # Download the $LYNIS_BASELINE_FILE ($lynis_baseline_file_bydefault) baseline
-    assert_script_run("wget --quiet " . data_url("lynis/$lynis_baseline_file") . " -O " . "$dir" . "$lynis_baseline_file");
+    #    assert_script_run("wget --quiet " . data_url("lynis/$lynis_baseline_file") . " -O " . "$dir" . "$lynis_baseline_file");
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit

See https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/CONTRIBUTING.md#coding-style for more details